src/platform/ESP32/ESP32Utils.cpp

```diff
@@ -321,16 +321,25 @@ CHIP_ERROR ESP32Utils::InitWiFiStack(void)
     }
 
 #if CHIP_DEVICE_CONFIG_ENABLE_WIFI_AP
-    if (!esp_netif_create_default_wifi_ap())
+    // Lets not create a default AP interface if already present
+    if (!esp_netif_get_handle_from_ifkey("WIFI_AP_DEF"))
```

Comment:
If possible, would recommend having a define for "WIFI_AP_DEF" at a single place and using that here.

Reply:
Defined constants for both and using wherever I found the occurrence of `WIFI_AP_DEF` and `WIFI_STA_DEF`